controller/test/src/SensorTests.cpp

```diff
@@ -0,0 +1,168 @@
+/* Copyright 2020, RespiraWorks
```

Comment:
WDYT about using the platformio CLI for tests?  platformio is our only documented build system, and it seems to have decent testing infrastructure.

https://docs.platformio.org/en/latest/plus/unit-testing.html

I was working on sending out a PR with some simple tests using that framework.

Reply:
No opinion, never used it before. I'll take a look tomorrow. I'm good to follow what folks think is good for test.

Reply:
> I'm good to follow what folks think is good for test.

FWIW I have zero opinions other than platformio is already set up and seems to work kind of ok.